src/main/java/edu/nextstep/raing/CarRacingInterface.java

```diff
@@ -0,0 +1,19 @@
+package edu.nextstep.raing;
+
+import edu.nextstep.raing.model.Car;
+
+import java.util.List;
+
+/**
+ * author       : gwonbyeong-yun <sksggg123>
+ * ------------------------------------------
+ * | email        : sksggg123               |
+ * | github       : github.com/sksggg123    |
+ * | blog         : sksggg123.github.io     |
+ * ------------------------------------------
+ * project      : java-racingcar
+ * create date  : 2019-06-15 23:39
+ */
+interface CarRacingInterface {
+    abstract List<Car> move();
```

Comment:
아래 코멘트에 대해 고민해봐요.
https://github.com/next-step/java-racingcar/pull/275#discussion_r294113134

```suggestion
    List<Car> move();
```

Reply:
“abstract”를 제거하는 것으로 피드백을 주셨는데 abstract는 interface에서 사용하지 않고 상속가능한 class에서 정의하는 방법으로 자식클래스에게 구현을 강제하는 용도로 사용하는게 올바르기에 제거가 필요한 부분으로 이해를 했는데 맞을까요?

Reply:
인터페이스의 메서드는 묵시적으로 public abstract입니다. 따라서 인터페이스의 `abstract` 키워드는 중복됩니다.